tests/integration/specific_executor_params/test_specific_params.py

```diff
@@ -0,0 +1,32 @@
+import copy
+
+from docarray import DocumentArray
+
+from jina import Executor, Flow, requests
+
+
+def test_specific_params():
+    class MyExec(Executor):
+        def __init__(self, params_awaited, *args, **kwargs):
+            super().__init__(*args, **kwargs)
+            self.params_awaited = params_awaited
+
+        @requests
+        def process(self, docs, parameters, **kwargs):
+            assert parameters == self.params_awaited
+
+    flow = (
+        Flow()
+        .add(uses=MyExec, name='exec1', uses_with={'params_awaited': {'key_1': True}})
+        .add(
+            uses=MyExec,
+            name='exec2',
+            uses_with={'params_awaited': {'key_1': True, 'key_2': False}},
+        )
+    )
+
+    with flow:
```

Comment:
I prefer the test to return the parameters received in some docs tags and assert on the client side.